account/manager_test.go

```diff
@@ -869,7 +869,7 @@ func TestAccountWithdrawal(t *testing.T) {
 	dustOutput := &wire.TxOut{Value: 0, PkScript: p2wsh}
 	_, _, err := h.manager.WithdrawAccount(
 		context.Background(), account.TraderKey.PubKey,
-		[]*wire.TxOut{dustOutput}, feeRate, bestHeight,
+		[]*wire.TxOut{dustOutput}, feeRate, bestHeight, 0,
```

Comment:
Should we also add a test case for a non-zero expiration that checks the account modifications are added correctly?

Reply:
totally agree. The tests are currently pretty coupled to mock/hard coded values so let's add it after #313 has been merged and we can manage the expected behavior better.